picard/script.py

```diff
@@ -475,57 +492,78 @@ def func_trim(parser, text, char=None):
         return text.strip()
 
 
-def func_add(parser, x, y):
-    """Add ``y`` to ``x``."""
+def func_add(parser, x, y, *args):
+    """Adds ``y`` to ``x``.
+       Can be used with arbitrary number of arguements. The result is
+       $add($add(x,y)...)
+    """
     try:
-        return str(int(x) + int(y))
+        return _compute('add', locals())
```

Comment:
You can pass them on the same way you receive them: `_compute('add', x, y, *args)` and receive them in `def _compute(operation, *args)` and use them as `args`.